components/brave_wallet_ui/page/screens/swap/hooks/useSwap.ts

```diff
@@ -191,7 +232,7 @@ export const useSwap = () => {
     fromAccount,
     fromToken,
     fromAmount: editingFromOrToAmount === 'from' ? fromAmount : '',
-    toAccountId,
+    toAccountId: toAccount?.accountId,
```

Comment:
Complimentary to my other remarks, sticking to `toAccountId` here is better.

Reply:
Changed!